project/Dependencies.scala

```diff
@@ -191,6 +191,11 @@ object Dependencies {
     "org.mockftpserver" % "MockFtpServer" % mockFtpServerV % Test
   )
 
+  val implDrsDependencies = List(
+    "com.typesafe.akka" %% "akka-http-spray-json" % akkaHttpV,
```

Comment:
On quick glance this seems like the wrong dependency to pulled in (plus all of its transitive dependencies). Does just `spray-json` work?

Reply:
Thanks for pointing it out! It actually is a wrong dependency to be pulled in! `spray-json` does work.